### ISSUE TITLE
Cannot use  the 1.3.1 version of aspectj plugin

### ISSUE THREAD

**Issue by trinitron01** [Archinamon/GradleAspectJ-Android#19]
Currently it is not possible to use the previous version of the plugin, compilation fails with :

```
Error:Could not find com.archinamon:android-gradle-aspectj:1.3.1.
Searched in the following locations:
    file:/C:/Program Files/Android/Android Studio/gradle/m2repository/com/archinamon/android-gradle-aspectj/1.3.1/android-gradle-aspectj-1.3.1.pom
    file:/C:/Program Files/Android/Android Studio/gradle/m2repository/com/archinamon/android-gradle-aspectj/1.3.1/android-gradle-aspectj-1.3.1.jar
    https://jcenter.bintray.com/com/archinamon/android-gradle-aspectj/1.3.1/android-gradle-aspectj-1.3.1.pom
    https://jcenter.bintray.com/com/archinamon/android-gradle-aspectj/1.3.1/android-gradle-aspectj-1.3.1.jar
    https://github.com/Archinamon/GradleAspectJ-Android/raw/master/com/archinamon/android-gradle-aspectj/1.3.1/android-gradle-aspectj-1.3.1.pom
    https://github.com/Archinamon/GradleAspectJ-Android/raw/master/com/archinamon/android-gradle-aspectj/1.3.1/android-gradle-aspectj-1.3.1.jar

```

The newest version with the new api seems to be incompatible with some projects, it would be good to have the 1.3.1 version still available and possible to use (maybe kept on the different branch)

**Comment by Archinamon** [comment]
Maybe I did miss the artifacts when merged 2.0, will take a look in two weeks. Sorry for inconvenience.

**Comment by Archinamon** [comment]
Reverted artifact deletion.
